src/package.json

```diff
@@ -6,8 +6,8 @@
     "typings": "index.d.ts",
     "nativescript": {
         "platforms": {
-            "android": "3.0.0",
-            "ios": "3.0.0"
+            "android": "3.1.1",
```

Comment:
Having platform version of the plugin set to 3.1.1 will mean that this plugin will not be compatible in 3.0 application just because of this setting. That's why we prefer to set 3.0.0 here.